stemlib/Latin/stemsrc/ls.nom

```diff
@@ -98038,6 +98045,10 @@ indu_mentum
 indu^pe^di^o		<orth>indu^pe^di_tus</orth>	<itype>a</itype>
 indu^pe^rans		<itype>antis</itype>
 indu^pe^ro		<orth type=alt>indu^pe^ra_tor</orth>
+indu^pe^ra_tor
```

Comment:
Anything line that doesn't start with
:le: or :no: or :vb: etc... is ignored by the analyzer (basically a comment) so you don't need to add this line.

Reply:
Line removed